lib/transformer/caster/template.rb

```diff
@@ -0,0 +1,28 @@
+module Transformer
+  module Caster
+    class Template
+      def initialize
+        @template = { area: {}, graph_values: {} }
+      end
+
+      def add_graph_value(key, method, value)
```

Comment:
Perhaps this could be simplified as `def add_graph_value(graph_method, value)` (where `graph_method` is  a`GraphMethod` instance)?